src/xrGame/alife_object.cpp

```diff
@@ -150,16 +169,15 @@ void CSE_ALifeObject::spawn_supplies(LPCSTR ini_string)
                     if (!j)
                         j = 1;
 
-                    bScope = nullptr != strstr(V, "scope");
-                    bSilencer = nullptr != strstr(V, "silencer");
-                    bLauncher = nullptr != strstr(V, "launcher");
+                    bScope = is_spawn_supplies_flag_set(V, "scope");
+                    bSilencer = is_spawn_supplies_flag_set(V, "silencer");
+                    bLauncher = is_spawn_supplies_flag_set(V, "launcher");
+
                     // probability
-                    if (nullptr != strstr(V, "prob="))
-                        p = static_cast<float>(atof(strstr(V, "prob=") + 5));
-                    if (fis_zero(p))
-                        p = 1.0f;
-                    if (nullptr != strstr(V, "cond="))
-                        f_cond = static_cast<float>(atof(strstr(V, "cond=") + 5));
+                    if (strstr(V, "prob=") != nullptr)
+                        sscanf(strstr(V, "prob=") + 5, "%f", &p);
```

Comment:
Sorry I was wrong here!
Initially, I though we can use sscanf like this: `prob=%f`, but it's not the case here, so original atof is better here.
```suggestion
                        p = static_cast<float>(atof(strstr(V, "prob=") + 5));
```

Reply:
Can you also allow edits from maintainers for your PR, so I can apply quick fixes like this?

Reply:
Could not find this option, added you write permissions for the fork just in case

Reply:
It should be on the right side, some in the bottom of the pull request bar (the one, where you see reviewers, assignees, labels, etc.), but ok.

Reply:
I bet it is related to the fact that fork is linked to organization instead of personal account :(
![image](https://github.com/user-attachments/assets/095e691a-79ad-4c23-b696-6b656b9e4f59)

Reply:
> I bet it is related to the fact that fork is linked to organization instead of personal account :( ![image](https://private-user-images.githubusercontent.com/18573775/411321784-095e691a-79ad-4c23-b696-6b656b9e4f59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxMTMwODUsIm5iZiI6MTczOTExMjc4NSwicGF0aCI6Ii8xODU3Mzc3NS80MTEzMjE3ODQtMDk1ZTY5MWEtNzlhZC00YzIzLWI2OTYtNmI2NTZiOWU0ZjU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA5VDE0NTMwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzMTE4YmY0YzE4NzE0NzQ2YTUyY2MzNzg0ZTBhZTQ0Y2VkY2VmZmYxMDYwNDdjNGYwZTFlMjcxZTY3MzMzMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.KFtMycFksrJBc7E13L9AXjD_TEjFahvQEtbrP8D7lvk)

I see. Seems the reason.